client/sites/hooks/test/use-transfer-with-software-start-mutation.tsx

```diff
@@ -86,15 +86,15 @@ describe( 'useRequestTransferWithSoftware', () => {
 		);
 	} );
 
-	it( 'should return an error if plugins or themes are not provided', async () => {
+	it( 'should return an error if both plugin_slug and theme_slug are not provided', async () => {
```

Comment:
Would it be worth having tests for these cases?

* A plugin is provided but not a theme.
* A theme is provided but not a plugin.